web/tests/libtest/env.py

```diff
@@ -346,7 +346,10 @@ def enable_auth(workspace):
     scfg_dict["authentication"]["enabled"] = True
     scfg_dict["authentication"]["method_dictionary"]["enabled"] = True
     scfg_dict["authentication"]["method_dictionary"]["auths"] = \
-        ["cc:test", "john:doe", "admin:admin123", "colon:my:password"]
+        ["cc:test", "john:doe", "admin:admin123", "colon:my:password",
+         "admin_group_user:admin123"]
+    scfg_dict["authentication"]["method_dictionary"]["groups"] = \
+        {"admin_group_user": ["admin_group"]}
```

Comment:
Shouldn't we add a test where the group is stored with lower case letters, but in the user configuration the group name contains upper case letters?

Reply:
Now I renamed the group permission name here to `admin_GROUP` and we add admin permission to `ADMIN_group`. I think it will be enough and there is no reason to add new extra test cases.